code/modules/vehicles/vehicle_key.dm

```diff
@@ -9,6 +9,12 @@
 	desc = "A keyring with a small steel key, and a rubber stun baton accessory."
 	icon_state = "keysec"
 
+/obj/item/key/security/suicide_act(mob/living/carbon/user)
+	user.visible_message("<span class='suicide'>[user] is putting \the [src] in [user.p_their()] ear and begins spinning! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	user.emote("spin")

```

Comment:
add a shame suicide if they're somehow unable to spin

Reply:
Can't say I know how to do that

Reply:
Got that figured out.